images/kubeadm/runner

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Copyright 2017 The Kubernetes Authors.
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
we should drop this runner entirely, we can use the entrypoint from the base image